docs/database/connect-to-existing-database.md

```diff
@@ -0,0 +1,366 @@
+
+## Add the Swagger user interface to the API project
+
+We'll use the Swagger UI to test the **AspireExistingDB.ApiService** project. Let's install and configure it:
+
+1. In Visual Studio, in the **Solution Explorer**, right-click the **AspireExistingDB.ApiService** project, and then select **Manage NuGet Packages**.
+1. Select the **Browse** tab, and then search for **Swashbuckle**.
+1. Select the **Swashbuckle.AspNetCore** package, and then select **Install**.
+1. If the **Preview Changes** dialog appears, select **Apply**.
+1. In the **License Acceptance** dialog, select **I Accept**.
+1. In the **AspireExistingDB.ApiService** project, open the **Program.cs** file.
+1. Locate the following line of code:
+
+    ```csharp
+    builder.Services.AddOpenApi();
+    ```
+
+1. Immediately after that line, add this line of code, which adds the Swagger generator in the dependency injection (DI) container:
+
+    ```csharp
+    builder.Services.AddSwaggerGen();
+    ```
+
+1. Locate the following lines of code:
+
+    ```csharp
+    if (app.Environment.IsDevelopment())
+    {
+        app.MapOpenApi();
+    }
+    ```
+
+1. Modify that code to match the following lines:
+
+    ```csharp
+    if (app.Environment.IsDevelopment())
+    {
+        app.MapOpenApi();
+        app.UseSwagger();
+        app.UseSwaggerUI();
+    }
+    ```
+
+## Configure a connection string in the App Host project
+
+Usually, when you create a cloud-native solution with .NET Aspire, you call the <xref:Aspire.Hosting.SqlServerBuilderExtensions.AddSqlServer*> method to initiate a container that runs the SQL Server instance. You pass that resource to other projects in your solution that need access to the database.
+
+In this case, however, you want to work with a pre-existing database outside of any container. There are three differences in the App Host project:
+
+- You don't need to install the `Aspire.Hosting.SqlServer` hosting integration.
+- You add a connection string in a configuration file, such as **appsetting.json**.
+- You call <xref:Aspire.Hosting.ParameterResourceBuilderExtensions.AddConnectionString*> to create a resource that you pass to other projects. Those projects use this resource to connect to the existing database.
+
+Let's implement that configuration:
+
+1. In Visual Studio, in the **AspireExistingDB.AppHost** project, open the **appsetting.json** file.
+1. Replace the entire contents of the file with the following code:
+
+    ```json
+    {
+        "ConnectionStrings": {
+            "sql": "Server=localhost;Trusted_Connection=True;TrustServerCertificate=True;Initial Catalog=WeatherForecasts;"
+        },
+        "Logging": {
+            "LogLevel": {
+                "Default": "Information",
+                "Microsoft.AspNetCore": "Warning",
+                "Aspire.Hosting.Dcp": "Warning"
+            }
+        }
+    }
+    ```
+
+1. In the **AspireExistingDB.AppHost** project, open the **Program.cs** file.
```

Comment:
Anything that refers to a file path or directory on disc should be italicized.

```suggestion
1. In the **AspireExistingDB.AppHost** project, open the _Program.cs_ file.
```

Reply:
Fixed.